plutus-contract/src/Plutus/Contract/Test/ContractModel/DoubleSatisfaction.hs

```diff
@@ -369,7 +368,7 @@ doubleSatisfactionCounterexamples dsc =
                                         & dsTx   .~ tx
   , let valueStolen0 = dsc & l . outAddress .~ stealerAddr
                            & dsTx . outputs %~ (withDatumOut:)
-                           & dsTx . inputs %~ (Set.insert newFakeTxIn)
+                           & dsTx . inputs %~ ((:) newFakeTxIn)
```

Comment:
this is a prepend right? can you use the same syntax as above?

Reply:
Indeed! Thanks!